apps/omg/lib/omg/state/transaction/validator.ex

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.State.Transaction.Validator do
```

Comment:
for me, this module is more related to Core than to Transaction

Reply:
But it's not, it processes transactions...

Reply:
It does **State**ful **Transacton** Validation - couldn't be placed more accurately :D